### PR TITLE
Documentation for Crowd

### PR DESCRIPTION
**Pull request by jjeongatl**
## Pull request description

- Missing documentation for Crowd
- Crowd clustering is not supported in helm chart. Removed in values.yaml

## Checklist
- [x] I have added unit tests
- [x] I have applied the change to all applicable products
- [x] The E2E test has passed (use `e2e` label)
- [x] (Atlassian only) Internal Bamboo CI is passing

### CODE REVIEW

File: docs/docs/userguide/CONFIGURATION.md [hunk 1]
```diff
@@ -255,6 +255,10 @@ By default, the Helm charts will not configure the products for Data Center clus
 
         Because of the limitations outlined under [Bamboo and clustering](../troubleshooting/LIMITATIONS.md#cluster-size) the `clustering` stanza is not available as a configurable property in the Bamboo `values.yaml`.
 
+    === "Crowd"
+
+        Clustering is enabled by default. To disable clustering, set crowd.clustering.enabled to false in ${CROWD_HOME}/shared/crowd.cfg.xml and rollout restart Crowd StatefulSet after the initial product setup is complete.
```
Comment (errcode1202) [inline docs/docs/userguide/CONFIGURATION.md:260]:
```suggestion
        Clustering is enabled by default. To disable clustering, set `crowd.clustering.enabled` to `false` in `${CROWD_HOME}/shared/crowd.cfg.xml` and rollout restart Crowd StatefulSet after the initial product setup is complete.
```

File: docs/docs/userguide/CONFIGURATION.md [hunk 2]
```diff
@@ -223,7 +223,7 @@ The Helm charts are not opinionated whether they have a [Kubernetes namespace](h
 If you wish, you can run multiple Helm releases of the same product in the same namespace.
 
 ## :fontawesome-solid-network-wired: Clustering
-By default, the Helm charts will not configure the products for Data Center clustering. In order to enable clustering, the `enabled` property for clustering must be set to `true`.
+By default, the Helm charts will not configure the products for Data Center clustering except Crowd. In order to enable clustering, the `enabled` property for clustering must be set to `true`.
```
Comment (errcode1202) [inline docs/docs/userguide/CONFIGURATION.md:226]:
```suggestion
By default, the Helm charts will not configure the products for Data Center clustering. In order to enable clustering, the `enabled` property for clustering must be set to `true`.

!!!info "Clustering by default for Crowd"
    Clustering is enabled by default for Crowd. To disable clustering, set `crowd.clustering.enabled` to `false` in `${CROWD_HOME}/shared/crowd.cfg.xml` and rollout restart Crowd StatefulSet after the initial product setup is complete.
```

**Review by errcode1202 — CHANGES_REQUESTED** [review]
Looks good, added some suggestions. Any reason why Crowd clustering is enabled by default?

File: docs/docs/userguide/CONFIGURATION.md (hunk 2 above)
Comment (jjeongatl) [reply docs/docs/userguide/CONFIGURATION.md:226]: This is explained in line 258

**Review by bianchi2 — APPROVED** [review]

File: docs/docs/userguide/CONFIGURATION.md (hunk 2 above)
Comment (jjeongatl) [reply docs/docs/userguide/CONFIGURATION.md]: @errcode1202 addressed in new commit

**Review by errcode1202 — APPROVED** [review]

**Pull request merged**